src/entry/crud/event-crud.service.js

```diff
@@ -148,7 +148,8 @@ function EventCrud(
    * @returns {Promise}
    */
   service.updateTypicalAgeRange = function(item) {
-    var path = item.apiUrl.origin + '/events/' + item.id;
+    var type = item.isEvent ? 'events' : 'places';
+    var path = item.apiUrl.origin + '/' + type + '/' + item.id;
```

Comment:
Nit: I think you can always use `item.apiUrl` instead of `item.apiUrl.origin + '/' + type + '/' + item.id`

Reply:
When I logged it it was pointed to `/event/ `and `/place/ `instead of `/events/` and `/places/`
That's why I needed to create it myself

Reply:
Ah yep the new endpoint probably only works on the plural paths